### PR TITLE
[Docs] Deployment on artifakt

### PR DESCRIPTION
**Pull request by AdamKasp**
| Q               | A
| --------------- | -----
| Branch?         | 1.10
| Bug fix?        | no
| New feature?    | yes

### CODE REVIEW

File: docs/cookbook/deployment/artifakt.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
+How to deploy Sylius to Artifact.com
+====================================
+
+1. What is Artifakt
+-------------------
+
+Artifakt is an all-in-one platform helping enterprises and developers fast-track deployments,
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/artifakt.rst:7]: Could you add a link to Artifakt's website here?

```suggestion
`Artifakt <https://www.artifakt.com/>`_ is an all-in-one platform helping enterprises and developers fast-track deployments,
```

File: docs/cookbook/deployment/artifakt.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+To get off to a good start with Artifakt, you first need to create a project.
+
+Click Create Project in the left menu and follow the steps to create your first project.
+The project code must be unique within the workspace, you can refer to projects for more information.
+
+.. image:: /_images/cookbook/deployment/artifakt/create_project.png
+    :scale: 55%
+    :align: center
+
+In the second step, select the Sylius runtime and version that corresponds to the application you want to deploy. Also choose the `region <https://docs.artifakt.com/platform/available-regions>`_ where your application and data will be deployed.
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/artifakt.rst:27]:
```suggestion
In the second step, select the Sylius runtime and version that corresponds to the application you want to deploy.
Also, choose the `region <https://docs.artifakt.com/platform/available-regions>`_ where your application and data will be deployed.
```

File: docs/cookbook/deployment/artifakt.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
+How to deploy Sylius to Artifact.com
```
Comment (lchrusciel) [inline docs/cookbook/deployment/artifakt.rst:1]:
```suggestion
How to deploy Sylius to Artifakt.com
```

File: docs/cookbook/deployment/artifakt.rst [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+2. How it works
+---------------
+
+Artifakt provides a service based on Cloud technologies to help build, deploy, monitor, secure, and manage the scalability of your applications.
+
+3. Create your First Project
+----------------------------
+
+To get off to a good start with Artifakt, you first need to create a project.
```
Comment (lchrusciel) [inline docs/cookbook/deployment/artifakt.rst:18]: Shouldn't we mention about account creation at this stage

File: docs/cookbook/deployment/artifakt.rst [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+manage and run complex web applications on enterprise-grade cloud infrastructures on a global scale.
+
+2. How it works
+---------------
+
+Artifakt provides a service based on Cloud technologies to help build, deploy, monitor, secure, and manage the scalability of your applications.
+
+.. hint::
+
+    If you don't have account on **Artifakt.com**, you should request a demo or contact support first.
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/artifakt.rst:17]: Could you move it under point `3.`?

File: docs/cookbook/deployment/artifakt.rst [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+When you create a project, you can use the default Artifakt code base for the chosen Sylius runtime, or you can link your own Git repository.
+If you wish to use the Artifakt default code base, simply click "Skip, and Create Project". Otherwise, you can choose the location of your repository and choose your preferred repository and default branch.
+If you did not link your Git repository during the project creation, you can do so afterwards by going to `Project → Settings → Environments` and clicking Link to a different repository so that you can then create environments using your application's source code.
+
+.. image:: /_images/cookbook/deployment/artifakt/github_settings.png
+    :scale: 55%
+    :align: center
+
+4. Creating your First Environment
+----------------------------------
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/artifakt.rst:42]:
```suggestion
4. Create an environment
------------------------
```

File: docs/cookbook/deployment/artifakt.rst [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+---------------
+
+Artifakt provides a service based on Cloud technologies to help build, deploy, monitor, secure, and manage the scalability of your applications.
+
+.. hint::
+
+    If you don't have account on **Artifakt.com**, you should request a demo or contact support first.
+
+3. Create your First Project
+----------------------------
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/artifakt.rst:20]:
```suggestion
3. Create an Artifakt project
-----------------------------
```

File: docs/cookbook/deployment/artifakt.rst [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+====================================
+
+1. What is Artifakt
+-------------------
+
+`Artifakt <https://www.artifakt.com/>`_ is an all-in-one platform helping enterprises and developers fast-track deployments,
+manage and run complex web applications on enterprise-grade cloud infrastructures on a global scale.
+
+2. How it works
+---------------
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/artifakt.rst:11]:
```suggestion
2. How does Artifakt work?
--------------------------
```

File: docs/cookbook/deployment/artifakt.rst [hunk 9]
```diff
@@ -0,0 +1,76 @@
+How to deploy Sylius to Artifakt.com
+====================================
+
+1. What is Artifakt
+-------------------
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/artifakt.rst:5]:
```suggestion
1. What is Artifakt?
--------------------
```

File: docs/cookbook/deployment/artifakt.rst [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+When creating an environment, you must define a name, the Git branch to deploy, the `type of platform <https://docs.artifakt.com/platform/platform-management>`_ to use, and the `criticality <https://docs.artifakt.com/platform/environment-lifecycle#environnement-et-criticite>`_ (environments can be either `critical` or `noncritical`).
+
+.. image:: /_images/cookbook/deployment/artifakt/create_environment.png
+    :scale: 55%
+    :align: center
+
+If you set up your Git repository at the project level, the branches associated with your repository are automatically retrieved and available in the drop-down list.
+
+5. Building your First Environment
+----------------------------------
```
Comment (CoderMaggie) [inline docs/cookbook/deployment/artifakt.rst:54]:
```suggestion
5. Build the environment
------------------------
```

**Review by CoderMaggie — CHANGES_REQUESTED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Adam! :tada:

**Pull request merged**